api/app/Policies/UserPolicy.php

```diff
@@ -188,12 +188,12 @@ protected function teamAbleToCheck(User $actor, string $roleId, string $teamId)
                 return $actor->isAbleTo('assign-any-teamRole');
             case 'process_operator':
                 // Community roles have the update-team-processOperatorMembership permission, and it should give them the ability to assign processOperator roles to pools in their community.
-                // TODO: uncomment this in issue #10364
-                // $pool = $team->teamable; // If we're adding a processOperator to a team, that team should represent a pool. Need validation?
-                // $community = $pool->community;
+                $pool = $team->teamable; // If we're adding a processOperator to a team, that team should represent a pool. Need validation?
```

Comment:
What I was originally thinking about with this comment about validation is that its _possible_ (though not likely) that the teamable is something other than a pool, in which case it might not have a community object and this could error. It might safest to do something like

```
$communityTeam = $team->teamable?->community?->team;
...
|| (isSet($communityTeam) && $actor->isAbleTo('update-team-processOperatorMembership', $communityTeam);)
```

Reply:
Aye aye, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11070/commits/29ce5bc57813e5529838a762226ae0ebef31f982